### ISSUE TITLE
Form notification input border contrast

### ISSUE THREAD

**Issue by ppvg** [minvws/nl-rdo-manon#432]
From independent accessibility report, SC 1.4.11:

<blockquote lang="nl">
Op pagina <a href="https://minvws.github.io/nl-rdo-manon/documentation/components/form-notification.html">components/form-notification.html</a> staan een aantal invoervelden. Er staat bijvoorbeeld een invoerveld met daarbij een foutmelding. Randen van invoervelden moeten ook voldoende contrast hebben, zodat ze goed te onderscheiden zijn. Dit is hier niet overal het geval. Bij de foutmelding is het contrast met het witte invoerveld 2,8:1. Bij de andere invoervelden is het contrast nog lager. In de huidige status staan er overal al teksten ingevuld in de invoervelden, maar waarschijnlijk zal dit normaal gesproken niet zo gedaan worden, en dan kan dit wel problemen opleveren.
</blockquote>

The documentation theme (`soft`) has insufficient contrast on the borders of input field with error notifications.
